sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -770,6 +871,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -dd
```

Comment:
I couldn't find this parameter. Does this exist?

Reply:
It is a typo from editing, will fix now. thanks